### PR TITLE
Added environment variables for web-api user/pass

### PR DESCRIPTION
**Pull request by mserranom**
Required after https://github.com/overleaf/web/commit/e1c35d30509174672bb2bd3ebb1501fb345ab41c#diff-0364f57fbff2fabbe941ed20c328ef1a

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -81,6 +81,15 @@ COPY ${baseDir}/init_scripts/ /etc/my_init.d/
 RUN cd /var/www && node git-revision > revisions.txt
 
 
+# Set Environment Variables
+# --------------------------------
+ENV WEB_API_USER "sharelatex"
+# password is regenerated in init_scripts/00_regen_sharelatex_secrets.sh
```
Comment (JuneKelly) [inline Dockerfile:87]: Will the regen script regenerate this password?

https://github.com/overleaf/docker-image/blob/master/init_scripts/00_regen_sharelatex_secrets.sh

From what I can see it looks for `CRYPTO_RANDOM` as a keyword to replace with the generated password, and only looks in the settings file?

**Review by JuneKelly — COMMENTED** [review]
One question.

File: Dockerfile (hunk 1 above)
Comment (mserranom) [reply Dockerfile:87]: You hint correctly, won't be regenerated. To an extent settings using those variables will be overriden due to https://github.com/overleaf/docker-image/blob/f92c6c27b03effc6aaae8aa77633b9ec492e3f57/settings.coffee#L6, but that might be not enough. Will do some extra thinking about it.

Comment (mserranom) [reply Dockerfile:87]: Ok, I checked the configurations and all password occurrences are being replaced by `CRYPTO_RANDOM`. There's no actual difference with previous releases of Server Pro, by setting `ENV WEB_API_PASSWORD "password"` we're just adding back what was removed in https://github.com/overleaf/web-internal/pull/2422

Comment (JuneKelly) [reply Dockerfile:87]: Right, so this is just setting a default, which will be (in practice) overwritten by the bootstrap scripts? sounds good.

**Review by JuneKelly — APPROVED** [review]

**Pull request merged**